# Methods for exRNA

## I. Imputation, Normalization and Batch Correction

* **Methods for Single Cell RNA-seq** 
  * \(1\) Dropout/Sparseness and Imputation
  * \(2\) Heterogeneity and Normalization
  * \(3\) Batch effect and Confounder
  * \(Pseudo-time and Others\)
* **Recommendations**:
  * 2020 _Nature COMMN_ - Embracing the **dropouts** in single-cell RNA-seq analysis
  * 2019 - _Nature Methods_ - A discriminative learning approach to differential expression analysis for single-cell RNA-seq
  * 2017 - _Nature Methods_ - Normalizing single-cell RNA sequencing data: challenges and opportunities

> [PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20single%20cell&mode=list)
>
> * \(1\) Dropout/Sparseness and Imputation
> * \(2\) Heterogeneity and Normalization
> * \(3\) Batch effect and Confounder
> * \(Pseudo-time and Others\)
>
> [Tutorial](https://lulab1.gitbook.io/training/part-iii.-case-studies/case-study-1.exrna-seq/1.4.normalization-issues)

## II.  Feature Selection Method

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20Feature%20Selection&mode=list)

## III. Network Approach and Clustering

* 2020 _Bioinformatics_ - Deeptype - Deep-learning approach to identifying cancer subtypes using high-dimensional genomic data
* 2018 _Nature COMMN._ - Pathway based subnetworks enable cross-disease biomarker discovery

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20network%20and%20clustering&mode=list)

## IV. Tumor Location Method

* [Methods for Metagenomics](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20metagenomics&mode=list): tumor locator \(tissue match\), ranking, etc
* [Methods for Tumor Location](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20locator&mode=list): tumor locator \(tissue match\), etc

