# Multi-modal

## I. LLMs for Multi-modal Data

* ✅ [**2024 Nature machine intelligence**](https://www.nature.com/articles/s42256-024-00868-w)  - A transformer-based weakly supervised computational pathology method for clinical-grade diagnosis and molecular marker discovery of gliomas
* **2024 Nature Methods** - scGPT: toward building a foundation model for single-cell multi-omics using generative AI
* ✅  **2023 Nature** - Foundation models for generalist medical artificial intelligence

## II. Deep Learning Models for Multi-modal Data

* [**2022 Nature Reviews**](https://www.nature.com/articles/s41576-022-00532-2) - Obtaining genetics insights from deep learning via explainable artificial intelligence
* **2022 Genome Biology** - A benchmark study of deep learning‑based multi‑omics data fusion methods for cancer
* ✅ **2022 Cancer Cell** - Pan-cancer integrative histology-genomic analysis via multimodal deep learning
* _**2022 Nature Biotech.**_ - Multi-omics single-cell data integration and regulatory inference with graph-linked embedding
* _**✨ 2021 Nature Communications**_ - MOGONET integrates multi-omics data using graph convolutional networks allowing patient classification and biomarker identification
* _**✨ 2021 Bioinformatics**_ - PathCNN: interpretable convolutional neural networks for survival prediction and pathway analysis applied to glioblastoma
* _**2021 Nature Machine Intelligence**_ - Integration of multiomics data with graph convolutional networks to identify new cancer genes and their associated molecular mechanisms
* _**2021 Bioinformatics -**_  Subtype-GAN: a deep learning approach for integrative cancer subtyping of multi-omics data
* _**2018 Clinical Cancer Research**_ - Deep Learning–Based Multi-Omics Integration Robustly Predicts Survival in Liver Cancer

## III. Other Models for Multi-modal Data

* **2024 Nature Communications** -  Enhancing NSCLC recurrence prediction with PET/CT habitat imaging, ctDNA, and integrative radiogenomics-blood insights
* **2023 Nature** - Transfer learning enables predictions in network biology
* **2022 BIB** - Blood-based transcriptomic signature panel identification for cancer diagnosis: benchmarking of feature extraction methods
* ✅ _**2022 Bioinformatics**_ - MOMA: a multi-task attention learning algorithm for multi-omics data interpretation and classification
* _**✨ 2022 BIB**_ - A computational framework to unify orthogonal information in DNA methylation and copy number aberrations in cell-free DNA for early cancer detection
* _**2019 Bioinformatics**_ - DIABLO: an integrative approach for identifying key molecular drivers from multi-omics assays
* _**2019 Bioinformatics**_ - Multiomics modeling of the immunome, transcriptome, microbiome, proteome and metabolome adaptations during human pregnancy



## IV. More Deep Learning Models

* **Deep Learning on RNA**
	* **3D Structure prediction of RNA**: 2021 _Science_ - Geometric deep learning of RNA structure
	* **2D structure** \(**Transfer learning**\): 2019 _Nature Commn._ - SPOT-RNA: RNA secondary structure prediction using an ensemble of two-dimensional deep neural networks and transfer learning
	* **RNA/DNA-Protein Binding (DeepBind)**: 2015 _Nature Biotech._ - DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning
	* **AS of RNA**: 2019 _Cell_ - Predicting Splicing from Primary Sequence with Deep Learning
	* **AS of RNA \(DARTS**\): 2019 _Nature Methods_ - Deep-learning augmented RNA-seq analysis of transcript splicing
	* **APA of RNA**: 2019 _Cell_ - A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation
* **Deep Learning on DNA**
	* **2022 Nature** - The evolution, evolvability and engineering of gene regulatory DNA
* **Deep Learning on Protein**
	* **2022 Nature Biotech.** - Using deep learning to annotate the protein universe
	* _**AlphaFold2**_ - **2021&#x20;**_**Nature**_ - Highly accurate protein structure prediction with AlphaFold
	* Baker et al. - **2021&#x20;**_**Science**_ - Accurate prediction of protein structures and interactions using a three-track neural network

