# RNA-Oligo

## I. siRNA/ASO

### 1. Efficacy prediction

* [_**Our papers: NAR 2008a, 2008b, 2008c**_](https://www.ncrnalab.org/publications/pub.html#ii-2-sirnashrna-targeting-rna)
* _**2018 BMC Genomics**_, SiRNA silencing efficacy prediction based on a deep architecture&#x20;
* _**2008 Bioinformatics**_, siDRM: an effective and generally applicable online siRNA design tool
* _**2007 NAR**_ - Comparison of approaches for rational siRNA design leading to a new efficient and transparent method
* _**2006 BMC Bioinformatics**_  - Computational models with thermodynamic and composition features improve siRNA design
* _**2006 BMC Bioinformatics**_ - An accurate and interpretable model for siRNA efficacy prediction
* _**2005 Nature Biotech.**_ - Design of a genome-wide siRNA library using an artificial neural network

### 2. Off-target

* ✅ [**2024 Bioinformatics** ](https://academic.oup.com/bioinformatics/article/40/1/btae011/7513164)- SeedMatchR: identify off-target effects mediated by siRNA seed regions in RNA-seq experiments
* ✅ _**2017 Cell Systems**_ - Explicit Modeling of siRNA-Dependent On- and Off-Target Repression Improves the Interpretation of Screening Results
* [**2013 Biomed Res Int.** ](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773410/) - SeedSeq: Off-Target Transcriptome Database

### 3. Modification & Toxicity

* **2023 Advanced Drug Delivery Reviews**  - A holistic analysis of the intrinsic and delivery-mediated toxicity of siRNA therapeutics
* **2020 Signal Transduction and Targeted Therapy** - Therapeutic siRNA: state of the art
* **2016 Scientific Reports** - siRNAmod: A database of experimentally validated chemically modified siRNAs

## **II. sgRNA**

### **1. sgRNA activity**&#x20;

* ✅ _**2018 Nature Biotech.**_ - Deep learning improves prediction of CRISPR–Cpf1 guide RNA activity
* [more sgRNA papers](https://portals.broadinstitute.org/gppx/crispick/public/faq)&#x20;

### **2a. sgRNA offtarget - computational methods**

* \[Benchmark] [**2023 BIB**](https://academic.oup.com/bib/article/24/6/bbad333/7286387) - Benchmarking deep learning methods for predicting CRISPR/Cas9 sgRNA on- and off-target activities
* \[review][ **2023 BIB**](https://academic.oup.com/bib/article/24/3/bbad131/7130974) - Using traditional machine learning and deep learning methods for on- and off-target prediction in CRISPR/Cas9: a review
* [**2023 Nature Communications** ](https://www.nature.com/articles/s41467-023-42695-4)- Genome-wide CRISPR off-target prediction and optimization using RNA-DNA interaction fingerprints
* ✅ \[off-target] _**2016 Nature Biotech.**_ -Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9

### **2b. sgRNA offtarget - experimental methods**

* **2024 Nat. Biotech.** - Tracking-seq reveals the heterogeneity of off-target effects in CRISPR–Cas9-mediated genome editing
* **2023 Nat. Methods** - Improving the sensitivity of in vivo CRISPR off-target detection with DISCOVER-Seq+
* **2019 Science** _-_ Unbiased detection of CRISPR off-targets in vivo using DISCOVER-Seq
